docker/docker-compose.yaml

```diff
@@ -162,6 +163,26 @@ services:
     networks:
       - von
 
+  cypress:
```

Comment:
Should this be in the docker-compose? I wouldn't normally expect to run this as part of running the demo

Reply:
mmm... the main reason why I used docker compose is because it does not require a package manager (NPM) or any modules, it just pulls a docker image. However came up with other idea:
- creeated a new script - `scripts/e2e.sh` which will install cypress and will run e2e tests, this is not ideal but for time being will cover us, ideally would like to include this as part of CI/CD